### PR TITLE
feat: [#188109500] Update error message when no license results are f…

### PR DESCRIPTION
**Pull request by cbrunefearless**
## Description

Acceptance Criteria
Given I have entered in a business name and address into the license status check task
and there is a problem with the system integration
when attempting to check my license status
then I am presented with the existing red alert "Unable to check, please try again later" error (see design for latest copy)

Given I have entered in a business name and address into the license status check task
when no licenses are found
then a message is displayed indicating that no license was found, instead of the saying "License could not be found..." (see design for latest copy)
and the alert uses the new yellow alert design

### Ticket

This pull request resolves [#188109500](https://www.pivotaltracker.com/story/show/188109500).

## Code author checklist

- [X] I have rebased this branch from the latest main branch
- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation on the engineering documentation website
- [X] I have not used any relative imports
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] I have checked for and removed instances of unused config from CMS
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [X] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/src/components/tasks/LicenseTask.tsx [hunk 1]
```diff
@@ -83,22 +83,22 @@ export const LicenseTask = (props: Props): ReactElement => {
...
         if (resultLicenseData.licenseStatus === "UNKNOWN") {
           analytics.event.task_address_form.response.fail_application_not_found();
           setError("NOT_FOUND");
         }
       })
-      .catch(() => {
-        setError("SEARCH_FAILED");
+      .catch((error) => {
+          setError("SEARCH_FAILED");
       })
```
Comment (cbrunefearless) [inline web/src/components/tasks/LicenseTask.tsx:102]: Lint error when error is there and not used, so I took it out..

File: web/src/components/tasks/CheckLicenseStatus.tsx [hunk 2]
```diff
@@ -101,12 +101,19 @@ export const CheckLicenseStatus = (props: Props): ReactElement => {
   const getErrorAlert = (): ReactElement => {
     if (!props.error) {
       return <></>;
+    } else if (props.error === "NOT_FOUND") {
```
Comment (SirSamuelJoseph) [inline web/src/components/tasks/CheckLicenseStatus.tsx:104]: Can we make the variant a ternary to reduce repeated code?

File: api/src/api/licenseStatusRouter.test.ts [hunk 3]
```diff
@@ -74,14 +74,13 @@ describe("licenseStatusRouter", () => {
...
-    const licenseTaskID = "some-task-id";
+    const licenseTaskID = "some-id";
     stubUserDataClient.get.mockResolvedValue(userData);
     const response = await request(app).post(`/license-status`).send({ nameAndAddress, licenseTaskID });
     expect(response.status).toEqual(StatusCodes.OK);
     expect(stubUserDataClient.get).toHaveBeenCalledWith("some-id");
     expect(stubUserDataClient.put).toHaveBeenCalledWith(userData);
     expect(response.body).toEqual(userData);
-    expect(stubUpdateLicenseStatus).toHaveBeenCalledWith(userData, nameAndAddress, licenseTaskID);
   });
```
Comment (SirSamuelJoseph) [inline api/src/api/licenseStatusRouter.test.ts:84]: Are we not expecting this to have been called anymore? Can't find the part of the code change related to that

Comment (cbrunefearless) [reply api/src/api/licenseStatusRouter.test.ts:85]: changed that

**Pull request merged**